apps/nextjs-website/src/app/auth/confirmation/page.tsx

```diff
@@ -36,6 +36,7 @@ const Confirmation = () => {
         })
         .catch((error) => {
           // TODO: remove console warn and handle errors: [CodeMismatchException, ExpiredCodeException, InternalErrorException, LimitExceededException]
+          // see apps/nextjs-website/src/app/auth/email-confirmation/page.tsx
```

Comment:
Why are you adding this comment here?

Reply:
I'm adding a detail for the future refactor